src/patternfly/components/Dropdown/dropdown-multi-select.hbs

```diff
@@ -5,22 +5,22 @@
   {{/if}}>
   <form class="pf-c-form">
   <fieldset class="pf-c-form__fieldset" aria-labelledby="{{id}}-label">
-    {{#> check check--modifier="pf-c-dropdown__menu-item"}}
+    {{#> check check--type="label" check--modifier="pf-c-dropdown__menu-item"}}
       {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-apples" name="fruit"')}}{{/check-input}}
-      {{#> check-label check-label--attribute=(concat 'for="' id '-apples"')}}Apples{{/check-label}}
+      {{#> check-label check-label--type="span" check-label--attribute=(concat 'for="' id '-apples"')}}Apples{{/check-label}}
```

Comment:
I think the `for=` should move up to the check, correct?

Reply:
doh! yep.

Reply:
double doh! this change isn't needed anymore, this example was removed :)